templates/repo/settings/options.tmpl

```diff
@@ -67,12 +67,20 @@
 
 		</div>
 
-		{{if .MirrorsEnabled}}
+		{{if and .MirrorsEnabled (or (or .PullMirrors .PushMirrors) .Repository.IsMirror)}}
```

Comment:
Potential logic error here: If mirrors are disabled, the Mirror Settings should not be shown to the user, even if their repository *was* a mirror, because it *doesn't serve the function of a mirror anymore*, if I understood this correctly.

It could be the case that mirrors are enabled, but no new push and pull mirrors are allowed and the repository is *not* a mirror. I think that the first `or` should be replaced with an `and`.

Reply:
It seems like a very trivial problem to fix, but I'll take a break for now.

Reply:
Wrong variable was used.